save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -62,13 +69,40 @@ data class GeneralConfig(
         } ?: this.tags
 
         return GeneralConfig(
-            this.execCmd,
+            this.execCmd ?: other.execCmd,
             mergedTag,
-            this.description,
-            this.suiteName,
+            this.description ?: other.description,
+            this.suiteName ?: other.suiteName,
             this.excludedTests ?: other.excludedTests,
             this.includedTests ?: other.includedTests,
             this.ignoreSaveComments ?: other.ignoreSaveComments
         )
     }
+
+    override fun validateAndSetDefaults(): GeneralConfig {
+        requireNotNull(execCmd) {
+            errorMsgForRequireCheck("execCmd")
+        }
+        requireNotNull(tags) {
+            errorMsgForRequireCheck("tags")
+        }
+        requireNotNull(description) {
+            errorMsgForRequireCheck("description")
+        }
+        requireNotNull(suiteName) {
+            errorMsgForRequireCheck("suiteName")
+        }
+        return GeneralConfig(
+            execCmd,
+            tags,
+            description,
+            suiteName,
+            excludedTests ?: "",
+            includedTests ?: "",
+            ignoreSaveComments ?: false
+        )
+    }
+
+    private fun errorMsgForRequireCheck(field: String) = "Error: Couldn't found `$field` in [general] section. Please provide it in this, " +
```

Comment:
I suppose that if it is easy - we should add a location of a test config with a problem

Reply:
Please take a look at https://github.com/cqfn/save/pull/116#discussion_r645447932